### ISSUE TITLE
Support "ALTER TABLE"

### ISSUE THREAD

**Issue by unclhos** [versionpress/versionpress#1151]
Some plugins alter the existing tables to add new information. These new columns are saved into the ini files already it seems. But the table schema is not updated in the .schema files. Could this be support and have the "apply-changes" command verify the table schema and update if needed so push/pulls will not error out with "to many fields" type of error.

**Comment by borekb** [comment]
I think this will generally be tricky to support. What do you think @JanVoracek?

**Comment by JanVoracek** [comment]
Yes, this will be quite difficult. Tracking is easy but undo / rollback / merge operations will need to handle a lot of "weird" situations.
